### PR TITLE
506 issue，add testCase for soul-admin ZookeeperDataChangedListener

### PR DESCRIPTION
**Pull request by tyjwan**
Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by tyjwan** [comment]
506 issue，add testCase for soul-admin ZookeeperDataChangedListener

File: soul-admin/src/main/resources/application-local.yml [hunk 1]
```diff
@@ -27,14 +27,13 @@ spring:
   datasource:
     url: jdbc:mysql://localhost:3306/soul?useUnicode=true&characterEncoding=utf-8
     username: root
-    password:
+    password: Twy118023
```
Comment (tydhot) [inline soul-admin/src/main/resources/application-local.yml:30]: don't modify here

File: soul-admin/src/test/java/org/dromara/soul/admin/listener/zookeeper/ZookeeperDataChangedListenerTest.java [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    public void testOnMetaDataChanged() {
+        String mockId = "MOCK_ID";
+        String mockAppName = "MOCK_APP_NAME";
+        String mockPath = "MOCK_PATH";
+        MetaData metaData = new MetaData();
+        metaData.setId(mockId);
+        metaData.setAppName(mockAppName);
+        metaData.setPath(mockPath);
+
+        MetaData metaData2 = new MetaData();
```
Comment (tydhot) [inline soul-admin/src/test/java/org/dromara/soul/admin/listener/zookeeper/ZookeeperDataChangedListenerTest.java:120]: don't name a variable this way

File: soul-admin/src/test/java/org/dromara/soul/admin/listener/zookeeper/ZookeeperDataChangedListenerTest.java [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+import java.net.URLEncoder;
+
+/**
+ * The testCase for ZookeeperDataChangedListener.
+ *
+ * @author tyjwan
+ * @date 2020/12/08
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class ZookeeperDataChangedListenerTest {
```
Comment (tydhot) [inline soul-admin/src/test/java/org/dromara/soul/admin/listener/zookeeper/ZookeeperDataChangedListenerTest.java:49]: final

File: soul-admin/src/test/java/org/dromara/soul/admin/listener/zookeeper/ZookeeperDataChangedListenerTest.java (hunk 2 above)
Comment (tyjwan) [reply soul-admin/src/test/java/org/dromara/soul/admin/listener/zookeeper/ZookeeperDataChangedListenerTest.java:120]: Thank you, I will modify , then submit

**Review by yu199195 — APPROVED** [review]

**Pull request merged**